source/_integrations/zha.markdown

```diff
@@ -583,6 +583,14 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
   - Place your Zigbee Coordinator away from any Wi-Fi access points and all other sources of WiFi.
   - Wi-Fi frequency ranges can overlap with Zigbee, see the section above on defining Zigbee channel use.
 
+### Problems upgrading Zigbee device firmware via OTA
+
+Be aware that OTAU (Over-The-Air Upgrade) of Zigbee devices typically takes around 10 minutes. While a device is updating a lot of traffic is generated on the network, therefore it is recommended to not execute multiple updates at the same time. Always check that battery-powered devices have at least 70% of battery-life left before starting a firmware upgrade (a general tip is therefore to simply put new fresh batteries in before upgrading any OTA firmware upgrade), OTA updating of firmware is very power-consuming and while some devices do check for a minimum battery-level before starting the upgrade and might refuse to update unless have enough battery-life left, not all do so.
+
+If Zigbee firmware upgrades do not start on a battery-powered product then be aware that you usually need to "wake up the device" so that will will be able to receive commands, (because battery-powered products are sleepy devices so they normally only receive commands when the state of the device is changed). If the upgrade still does not start then try restarting the device by disconnecting the power/battery for a few seconds and try again, again make sure to activate the device by pressing a button on it right before sending the update request. Try to push a button on the device (if it has one) shortly before you start the firmware update to wake it up. Sometimes it also helps to try keeping the device awake by pushing a button repeatedly until you see the first "Updating... " message in the user interface.
```

Comment:
```suggestion
If Zigbee firmware upgrades do not start on a battery-powered product, then be aware that you usually need to "wake up the device" so that it will be able to receive commands (because battery-powered products are sleepy devices, so they normally only receive commands when the state of the device is changed). If the upgrade still does not start, then try restarting the device by disconnecting the power/battery for a few seconds and try again; make sure to activate the device by pressing a button on it right before sending the update request. Try to push a button on the device (if it has one) shortly before you start the firmware update to wake it up. Sometimes, it also helps to try keeping the device awake by repeatedly pushing a button until you see the first "Updating... " message in the user interface.
```